json-rpc/types/src/proto/jsonrpc.proto

```diff
@@ -146,13 +146,6 @@ message EventData {
    * for more details
    */
   uint64 role_id = 19 [json_name="role_id"];
-
-  /**
-   * admintransaction event field.
-   * The block time when this transaction is committed.
-   * It is created by validators.
-   */
-  uint64 committed_timestamp_secs = 20 [json_name="committed_timestamp_secs"];
```

Comment:
why delete this?

Reply:
```
    #[serde(rename = "admintransaction")]
    AdminTransaction { committed_timestamp_secs: u64 },

```
is still there

Reply:
Ah that was probably a typo. Lemme fix this.